docs/requirements.txt

```diff
@@ -1,2 +1 @@
-breathe
-# git+git://github.com/michaeljones/breathe@v4.3.1#egg=breathe
+exhale
```

Comment:
To build locally, you'll need to `pip3 install exhale`.  NOTE: py3 is noticeably faster than py2 ;)

Reply:
Nice to hear -- many times, it is unfortunately the other way ;).